assets/js/localization.js

```diff
@@ -100,7 +100,7 @@ $(document).ready(function () {
 
         var pathname /*: string */ = window.location.pathname;
 
-        if(window.history.replaceState && !pathname.endsWith('/index.debug.html')) {
+        if(window.history.replaceState && !(pathname.indexOf('/index.debug.html') === pathname.length - '/index.debug.html'.length)) {
```

Comment:
what's going on here?

Reply:
@sushain97 This is because phantomJS, where we run our continuous integration tests for QUnit, isn't on ES6. This is the simple fix without importing other packages.

Reply:
Doesn't compat.js have shims? It could be included in the test head to fix it.